docs/02.API-REFERENCE.md

```diff
@@ -3691,6 +3710,66 @@ jerry_resolve_or_reject_promise (jerry_value_t promise,
 
 These APIs all depend on the es.next profile (or on build options).
 
+## jerry_get_well_known_symbol
+
+**Summary**
+
+Get the well-known symbol corresponds to the given [well-known symbol id](#jerry_well_known_symbol_t).
```

Comment:
Maybe: `... symbol corresponding to the given ... ` ?

Reply:
Thanks, I've fixed this misspelling.